pkg/document/crdt/tree.go

```diff
@@ -1030,6 +1030,19 @@ func (t *Tree) ToIndex(parentNode, leftSiblingNode *TreeNode) (int, error) {
 	return idx, nil
 }
 
+// ToPath returns path from given CRDTTreePos
+func (t *Tree) ToPath(parentNode, leftSiblingNode *TreeNode) ([]int, error) {
+	treePos, err := t.toTreePos(parentNode, leftSiblingNode)
+	if err != nil {
+		return make([]int, 0), err
```

Comment:
```suggestion
		return nil, err
```

https://github.com/uber-go/guide/blob/master/style.md#nil-is-a-valid-slice

Reply:
Thanks for suggestion, I will reflect the corrections according to the guide you mentioned.